digitaloceanmodule.php

```diff
@@ -2,25 +2,11 @@
 
 class Digitaloceanmodule extends Module {
 
-    private static $version = "2.2";
-    private static $authors = array(array('name' => "ModulesBakery.com.", 'url' => "http://www.modulesbakery.com"));
-
     public function __construct() {
         Loader::loadComponents($this, array("Input"));
         Language::loadLang("lang", null, dirname(__FILE__) . DS . "language" . DS);
         Loader::loadHelpers($this, array("Html"));
-    }
-
-    public function getName() {
-        return Language::_("Digitaloceanmodule.name", true);
-    }
-
-    public function getVersion() {
-        return self::$version;
-    }
-
-    public function getAuthors() {
-        return self::$authors;
+        $this->loadConfig(dirname(__FILE__) . DS . "composer.json");
```

Comment:
Why are you loading a composer file as the module config file? A Blesta config file (https://docs.blesta.com/display/dev/Module+Configuration) and a composer file are very different.

You need to include a config.json file with the appropriate information as described in the link above, or you need to define the required methods. 

I would typically expect separate pull requests for something like this--one to convert the module to use the JSON config, and another for each specific fix to the module.

I see the module is currently on version "2.2", which I assume is semantically "2.2.0". A change should also increment the version to "2.2.1", since unfortunately, the module defines its own version separate from the package itself.